src/styles/blocks/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  width: $card-width;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  &__product-image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__product-title {
+    font-weight: $medium-font;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 0 4px;
+  }
+
+  &__product-code {
+    color: $secondary-text-color;
+    margin: 0 0 16px;
+  }
+
+  &__rating-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin: 0 0 24px;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-label {
+    color: $secondary-text-color;
+    font-size: 12px;
+    line-height: 16px;
+  }
+
+  &__price-amount {
+    font-weight: $bold-font;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__buy-link {
+    height: 40px;
+    border-radius: 5px;
+
+    background-color: $main-color;
+    text-transform: uppercase;
+    text-align: center;
+    text-decoration: none;
+    font-weight: $bold-font;
+    color: $main-contrast-color;
+    font-size: 14px;
+    line-height: 16px;
+
+    display: flex;
+    box-sizing: border-box;
+    justify-content: center;
+    align-items: center;
+
+    &:hover {
+      background-color: $main-contrast-color;
+      border: solid 1px $main-color;
+      color: $main-color;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
+  }
```

Comment:
The hover effect with `transform: scale(1.2);` can cause layout shifts which might not be desirable. It's important to ensure that the scaling does not affect the surrounding elements. If the design allows, consider scaling only the image or content inside the card rather than the entire card.